scripts/infra/cloud-instance.sh

```diff
@@ -20,9 +20,9 @@ ec2() {
         exit 1
     fi
     shift
-    if [ "$(type -t "ec2__$cmdname")" = "function" ] >/dev/null 2>&1; then
+    if [ "$(type -t "ec2__${cmdname//-/_}")" = "function" ] >/dev/null 2>&1; then
```

Comment:
This wouldn't be necessary if the method was done for all cloud types.  If we do want to make this change, should also change it for ibm()

Reply:
ok - changing to support both

Reply:
So this change isn't needed since update_ssh_config is now generic. But if we want this to go in for the future, just need to make the corresponding change in ibm().

Reply:
sorry, i thought i removed this. i think I lost it in a rebase

Reply:
removed it for real this time!